src/main.cpp

```diff
@@ -228,7 +228,14 @@ static int RunStop(int argc, const char* argv[])
         printf("%sERROR: failed to write result file '%s'.%s\n", col::kRed, outFile.c_str(), col::kReset);
         return 1;
     }
-    fwrite(bigJson.data(), 1, bigJson.size(), fout);
+    const size_t numBytesWritten = fwrite(bigJson.data(), 1, bigJson.size(), fout);
+    if (numBytesWritten != bigJson.size())
+    {
+        printf("%sERROR: failed to write result file '%s', %llu of %llu bytes written.%s\n",
```

Comment:
Minor: I think `%zu` is a widely working format arg for `size_t` types, without need for extra casts.